### PR TITLE
Adding ONAP catalog (Open Network Automation Platform)

### PR DESCRIPTION
**Pull request by gbarsky**

### CODE REVIEW

**Comment by gbarsky** [comment]
Hello, 
I would appreciate any help to figure out why pull request is failing .

Thanks

File: templates/onap/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.catalog:
+  name: "ONAP"
+  version: "v1.1"
```
Comment (rawmind0) [inline templates/onap/0/rancher-compose.yml]: Version value here has to match with `templates/onap/config.yml` version value.

File: templates/onap/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: ONAP	
+description: |
+  ONAP is a comprehensive platform for real-time, policy-driven orchestration and automation of physical and virtual network functions that will enable software, network, IT and cloud providers and developers to rapidly create new services.
+version: 1.1
```
Comment (rawmind0) [inline templates/onap/config.yml]: Version value here has to match with `templates/onap/0/rancher-compose.yml` version value.

**Review by rawmind0 — CHANGES_REQUESTED** [review]

**Comment by rawmind0** [comment]
Hi @gbarsky , thank for the PR and contribute.

Build is failing due to version value is not matching. Take a look to the change requested.

**Comment by rawmind0** [comment]
Duplicated PR https://github.com/rancher/community-catalog/pull/674

**Comment by cjellick** [comment]
Very stale. Closing. Thanks! Please open a new PR if there is still a relevant change.

**Pull request closed without merging**